Source/WebCore/html/canvas/GPUCanvasContext.h

```diff
@@ -138,6 +138,10 @@ class GPUCanvasContext final : public GPUBasedCanvasRenderingContext {
     Ref<DisplayBufferDisplayDelegate> m_layerContentsDisplayDelegate;
     RefPtr<GPUSwapChain> m_swapChain;
     RefPtr<GPUSurface> m_surface;
+    struct GPUTextureHolder {
+        Ref<GPUTexture> texture;
+    };
+    std::unique_ptr<GPUTextureHolder> m_textureHolder;
```

Comment:
I don't understand. Why do we need a texture holder?

I think this lifetime management of render buffers should be done in a separate patch; it's a distinct problem from "nothing is getting destroyed ever."

Also, I'm making a lot of changes in this area; we should coordinate to make sure we don't step on each others' toes.